lib/tbot/config/testdata/TestTemplateSSHClient_Render/latest_OpenSSH/ssh_config.golden

```diff
@@ -21,6 +21,6 @@ Host *.tele.aperture.labs tele.blackmesa.gov
 # Flags for all tele.aperture.labs hosts except the proxy
 Host *.tele.aperture.labs !tele.blackmesa.gov
     Port 3022
-    ProxyCommand "/path/to/tbot" proxy --destination-dir=/test/dir --proxy-server=tele.blackmesa.gov:443 ssh --cluster=tele.aperture.labs  %r@%h:%p
+    ProxyCommand "/path/to/tbot" ssh-proxy-command --destination-dir=/test/dir --proxy-server=tele.blackmesa.gov:443 --cluster=tele.aperture.labs --tls-routing=true --connection-upgrade=false --resume=true --user=%r --host=%h --port=%p
```

Comment:
@rosstimothy / @espadolini  - double check that tI'm not missing anything here.

Reply:
Yeah right now the answer is "Manually modify the file" - I wanted to keep this PR a manageable size. What I'll do in another PR is allow them to be configured directly within the `tbot.yaml` and then they will be written to `proxytemplates.yaml` in the output destination.